### PR TITLE
Edit components

### PR DESCRIPTION
**Pull request by Bandius**
Added edit components for Tenant, User and Room

### CODE REVIEW

File: frontend/admin/src/components/Tenant/TenantEdit.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { EditProps, Edit, SimpleForm, TextInput, required } from 'react-admin'
+
+const TenantEdit: (props: EditProps) => JSX.Element = (props) => (
+  <Edit {...props}>
+    <SimpleForm>
+      <TextInput source="name" validate={required()} />
+      <TextInput source="email" validate={required()} />
```
Comment (matusziak) [inline frontend/admin/src/components/Tenant/TenantEdit.tsx:8]: We currently don't have an email field for this in the models. Does it make sense from some specific use case perspective I am not thinking of to add it? If not please remove it.

**Review by matusziak — APPROVED** [review]
image

I suggest that if you click on a row of an admin (user) it should take you to that admin's detail page.

Otherwise LGTM, please see the comments, fix & merge.

File: frontend/admin/src/components/Tenant/TenantEdit.tsx (hunk 1 above)
Comment (matusziak) [reply frontend/admin/src/components/Tenant/TenantEdit.tsx:8]: I noticed, that Tenant create has email and phone. I think we could remove them for now for the sake of simplicity.

Comment (Bandius) [reply frontend/admin/src/components/Tenant/TenantEdit.tsx]: Removed fields, as requested.

**Pull request merged**